cs/src/core/VarLen/SpanByteFunctions.cs

```diff
@@ -10,19 +10,11 @@ namespace FASTER.core
     /// </summary>
     public class SpanByteFunctions<Key, Output, Context> : FunctionsBase<Key, SpanByte, SpanByte, Output, Context>
     {
-        /// <summary>
-        /// Whether we lock values before concurrent operations
-        /// </summary>
-        protected readonly bool locking;
-
         /// <summary>
         /// Constructor
         /// </summary>
         /// <param name="locking"></param>
-        public SpanByteFunctions(bool locking = false)
-        {
-            this.locking = locking;
-        }
+        public SpanByteFunctions(bool locking = false) : base(locking) { }
```

Comment:
Why are we still doing locking inside ConcurrentWriter below? Shouldn't it be refactored into Lock() and Unlock() overrides.